crates/ty/Cargo.toml

```diff
@@ -5,7 +5,8 @@ edition.workspace = true
 rust-version.workspace = true
 homepage.workspace = true
 documentation.workspace = true
```

Comment:
We can do this as a separate PR but we could also override homepage and documentation with `"https://github.com/astral-sh/ty/"`